src/ItemGrid.h

```diff
@@ -33,12 +34,10 @@ class cItemGrid
 
 	cItemGrid(int a_Width, int a_Height);
 
-	~cItemGrid();
-
 	// tolua_begin
 	int GetWidth   (void) const { return m_Width; }
 	int GetHeight  (void) const { return m_Height; }
-	int GetNumSlots(void) const { return m_NumSlots; }
+	int GetNumSlots(void) const { return static_cast<int>(m_Slots.size()); }
```

Comment:
What does this cast do? size_type is aliased to int, so this would be casting from int to int.

Reply:
Removed.